dev/tools/localization/bin/gen_missing_localizations.dart

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This program updates the language locale arb files with any missing resource
+// entries that are included in the english arb files. This is useful when
+// adding new resources for localization. You can just add the appropriate
+// entries to the english arb file and then run this script. It will then check
+// all of the other language locale arb files and update them with new 'TBD'
+// entries for any missing resources. These will be picked up by the localization
+// team and then translated.
+//
+// ## Usage
```

Comment:
Since this isn't a doc comment, markdown isn't needed.

Reply:
Good point. I just cribbed this from another example.